lib/k6-summary/0.0.1/index.js

```diff
@@ -299,7 +299,11 @@ function summarizeMetrics(options, data, decorate) {
     }
   })
 
-  names.sort()
+  // sort all metrics but keep submetrics with their main metric
+  names.sort(function (a, b) {
+    // SPACE is the first non-control ascii character
+    return a.replace('{', ' ').localeCompare(b.replace('{', ' '))
```

Comment:
Not sure if `localeCompare` is the way to go here considering <https://stackoverflow.com/questions/14677060/400x-sorting-speedup-by-switching-a-localecompareb-to-ab-1ab10> (but how many metrics do we expect in practice?)